<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SamplerWindow</class>
 <widget class="QMainWindow" name="SamplerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SamplerWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="prevSetBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>121</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextSetBtn">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>90</y>
      <width>121</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sample0Btn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Sample 0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sample1Btn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Sample 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sample2Btn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Sample 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sample3Btn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Sample 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sample4Btn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Sample 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sample6Btn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>180</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Sample 6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sample7Btn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>180</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Sample 7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sample5Btn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Sample 5</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sample0Box">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLCDNumber" name="setLcdNumber">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QComboBox" name="sample1Box">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sample2Box">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>140</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sample3Box">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>140</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sample4Box">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>290</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sample5Box">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>290</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sample7Box">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>290</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sample6Box">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>290</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
